deploy/packer/app.json

```diff
@@ -4,8 +4,8 @@
   "variables": {
     "npmPackage":          "",
     "templateContents":    "",
-    "hvmSourceAMI":        "ami-0710db67",
-    "pvSourceAMI":         "ami-ee17dc8e",
+    "hvmSourceAMI":        "ami-7c22f71c",
+    "pvSourceAMI":         "ami-e422f784",
```

Comment:
yup, this was in the base.json in my cot branch:
    {
      "type": "shell",
      "inline": [
        "sudo bash -c 'echo net.ipv4.tcp_challenge_ack_limit = 999999999 >> /etc/sysctl.conf'"
      ]
    }